src/wp-admin/includes/upgrade.php

```diff
@@ -2649,6 +2649,32 @@ function drop_index( $table, $index ) {
 	return true;
 }
 
+/**
+ * Checks if an index exists for a specified column in a table.
+ *
+ * @since 6.8.0
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param string $table  The name of the table.
+ * @param string $column The name of the column to check for an index.
+ * @return bool True if the index exists, false otherwise.
```

Comment:
```suggestion
 * @param string $column The name of the column to check for an index.
 *
 * @return bool True if the index exists, false otherwise.
```

Not sure wether this is a rule, but iirc there is always one blank line between param and return.

Reply:
Fixed